packages/workbox-webpack-plugin/src/lib/generate-webpack-manifest.js

```diff
@@ -0,0 +1,4 @@
+module.exports = (manifestEntries, manifestVarName) =>
```

Comment:
Nit: This file should probably be named something else. `generate-webpack-manifest` makes it sound like a webpack thing rather than a file manifest that is used in the service worker.

Reply:
> generate-manifest-with-webpack

This works!